source/octf/proto/InterfaceTraceManagement.proto

```diff
@@ -14,10 +14,25 @@ message TracePathPrefix {
     string prefix = 1 [
         (opts_param).cli_required = false,
         (opts_param).cli_long_key = "prefix",
-        (opts_param).cli_desc =
-            "Prefix of trace path, used for specifying a set of traces. "
-            "Used as a complete trace path or as path prefix when ended with "
-            "'*'"
+        (opts_param).cli_desc = "Prefix of trace path used as a complete trace "
+                                "path or as path prefix when ended with '*'"
+    ];
+}
+
+message RemoveTracesRequest {
+    string prefix = 1 [
+        (opts_param).cli_required = false,
+        (opts_param).cli_long_key = "prefix",
+        (opts_param).cli_short_key = "p",
+        (opts_param).cli_desc = "Prefix of trace path used as a complete trace "
+                                "path or as path prefix when ended with '*'"
+    ];
+
+    bool force = 2 [
+        (opts_param).cli_required = false,
+        (opts_param).cli_long_key = "force",
+        (opts_param).cli_short_key = "f",
+        (opts_param).cli_desc = "Enforce removing specified trace(s)"
```

Comment:
I would describe it's practical use: "Enforces removing specified traces in any state"

Additionally it may be worth to mention that only Error and Completed traces are removed by default (in command or flag description )